test/ogc-api-coverages/get-coverage-rangeset.ts

```diff
@@ -198,6 +198,44 @@ describe('OGC API Coverages - getCoverageRangeset', function () {
         expect(this.res.status).to.be.lessThan(400);
       });
     });
+
+    describe('which contains both form and query parameter', function () {
+      const queryLocal = { ...query };
+      delete queryLocal.subset;
+      const queryParameterString = 'subset=time%28%222020-01-02T00%3A00%3A00Z%22%3A%222020-01-02T01%3A00%3A00Z%22%29';
+      StubService.hook({ params: { redirect: 'http://example.com' } });
+      hookPostRangesetRequest(
+        version,
+        collection,
+        variableName,
+        queryLocal,
+        queryParameterString,
+      );
+
+      it('successfully queries CMR and accepts the request', function () {
```

Comment:
This check would not have been failed due to the original bug. I think you should add one more check that the temporal range is included in what gets passed into the backend service.

Reply:
Something like these checks:
```
      it('passes the temporal range to the backend service', function () {
        const { start, end } = this.service.operation.temporal;
        expect(start).to.equal('2020-01-02T00:00:00.000Z');
        expect(end).to.equal('2020-01-02T01:00:00.000Z');
      });

      it('identifies the correct granule based on time range', function () {
        const source = this.service.operation.sources[0];
        expect(source.granules.length === 1);
        expect(source.granules[0].id).to.equal('G1233800352-EEDTEST');
      });
```

Reply:
Yes we should make sure this info is passed to backend successfully ... I added your suggestions (I don't think I can improve that:).